packages/webapp/src/containers/SensorReadings/index.jsx

```diff
@@ -50,6 +53,35 @@ function SensorReadings({ history, match }) {
     }
   }, [readingTypes, location_id]);
 
+  const forecastInfo = useMemo(() => {
```

Comment:
What do you think about a new component for this?

Reply:
I was not so sure if this was worth a file!
But since you mentioned, I'm comfortable creating a component 😁